### ISSUE TITLE
How to reproduce your best result?

### ISSUE THREAD

**Issue by Xiangyu-CAS** [Qidian213/Ranked_Person_ReID#9]
Hi, I was trying to reproduce your best result (mAP=88.7%) by simply runing 
```
python tools/train.py --config_file='configs/softmax_ranked.yml' DATASETS.NAMES "('market1501')" 
```
but I got a much worse result at mAP=86.5%, with 2% margin to claimed result. Moreover, it's even lower than using triplet loss.  What I can do to get your claimed result?

| model| mAP  | loss |
| :------: |  :------: | :------: |
| r50-ibn-a(my result)|87.1%| softmax + triplet|
| r50-ibn-a(my result)|86.5%| softmax + ranked|
| r50-ibn-a(your result)|88.7%| softmax + ranked|

Thank you!

**Comment by Qidian213** [comment]
Did you change training paramters ? 
There are some exp result, you can refer (Resnet50_ibn_a).
Try more epoch ?

`

w | r | T | DukeMTMC-ReID | Market1501
-- | -- | -- | -- | --
mAP | rank1 | mAP | rank1
0.2 | 0.7 | 1.0 | 80.2 | 90.1 | 88.7 | 95.3
0.4 | 0.7 | 1.0 | 80.8 | 90.9 | 88.9 | 94.9
0.6 | 0.7 | 1.0 | 80.6 | 90.2 | 89.0 | 95.1

`
`

w | r | T | DukeMTMC-ReID | Market1501
-- | -- | -- | -- | --
mAP | rank1 | mAP | rank1
0.4 | 0.6 | 1.0 | 80.3 | 90.6 | 88.7 | 94.6
0.4 | 0.7 | 1.0 | 80.8 | 90.9 | 88.9 | 94.9
0.4 | 0.8 | 1.0 | 80.8 | 90.4 | 88.8 | 94.8

`
`

w | r | T | DukeMTMC-ReID | Market1501
-- | -- | -- | -- | --
mAP | rank1 | mAP | rank1
0.4 | 0.7 | 0.5 | 80.4 | 90.6 | 88.7 | 95.1
0.4 | 0.7 | 1.0 | 80.8 | 90.9 | 88.9 | 94.9
0.4 | 0.7 | 5.0 | 80.3 | 90.6 | 88.5 | 94.7

`

**Comment by say2sarwar** [comment]
I used your RankedListLoss for car196 dateset but i can't reproduce the results. the loss is increasing with every epoch. any thoughts?

**Comment by Mr-Da-Yang** [comment]
![O@_@$Y6D%_LO(12G_9J6@QD](https://user-images.githubusercontent.com/58104855/111569686-277e2500-87de-11eb-8a68-5a2a6171695f.png)
Hello, what is the meaning of these hyperparameters？
Looking forward to your reply！thank you
